cli/flox/src/commands/pull.rs

```diff
@@ -356,12 +357,10 @@ impl Pull {
                             Use 'flox edit' to address issues before activating.
                         "};
 
-                        message::created(message_with_warning);
+                        Ok(message::Action::Warning(message_with_warning))
```

Comment:
```suggestion
                        Ok(message::Action::Created(message_with_warning))
```

Reply:
i know it says warning, not sure if changing the message type was intentional though